<launch>

    <!-- Load Parametrs for MQTT -->
    <rosparam file ="$(find pkg_ros_iot_bridge)/config/config_pyiot.yaml"/>

    <!-- Action Server for ROS IOT Bridge -->
    <node pkg="pkg_ros_iot_bridge" name="node_action_server_ros_iot_bridge" type="node_action_server_ros_iot_bridge.py" output="screen"></node>



</launch>